lib/nrf_modem_lib/Kconfig.modemlib

```diff
@@ -77,6 +77,27 @@ config NRF_MODEM_LIB_TRACE_THREAD_PRIO
 	depends on NRF_MODEM_LIB_TRACE_THREAD_PRIO_OVERRIDE
 	default 0
 
+config NRF_MODEM_LIB_TRACE_BITRATE
+	bool "Enable trace backend bitrate measurement"
```

Comment:
What about "Measure trace backend bitrate"

Reply:
I prefer this because it mirrors the description for the tracing Kconfig option.

Reply:
I think we should avoid "Enable" in the KConfig text since it appears next to a checkbox, and so it is implied that checking the box enables the feature.